config/config.exs

```diff
@@ -88,6 +90,14 @@ config :skate, SkateWeb.AuthManager,
   issuer: "skate",
   secret_key: nil
 
+config :skate, Skate.Repo,
+  database: "skate_dev",
+  username: System.get_env("POSTGRES_USERNAME", System.get_env("USER")),
+  password: System.get_env("POSTGRES_PASSWORD", ""),
+  hostname: System.get_env("POSTGRES_HOSTNAME", "localhost"),
+  port: System.get_env("POSTGRES_PORT", "5432") |> String.to_integer(),
```

Comment:
Will these be read at run time or build time? I thought that this would need to do the `{:system, "POSTGRES_USERNAME"}` thing like the other env variables in this file, but I see Glides' database config calls `System.get_env()`, so now I'm not sure. If it was `{:system,}`, would that even work? Would it be able to remove the need for the `add_prod_credentials` function? Should prod get these from secrets manager instead of ENV variables?

Reply:
I think we can actually use `:system` now, it was causing errors in an earlier iteration of the code but it looks to be OK now. Will push update assuming CI is good. One thing about that is that henceforth we'll have to do `env \`cat .env\` mix test`.

We do still need the `add_prod_credentials` though, and (most of) the prod credentials do belong in environment variables. As I discussed with Ian, and feel free to correct me if I'm wrong @ianwestcott, of the credentials the only secret is the password. But that password's not a secret that lives in Secrets Manager: it's a temporary password that we fetch through the RDS API (that's what that `auth_token_fn` is about in `add_prod_credentials`), and fetching that is really `add_prod_credentials`' main job.

Reply:
@skyqrose actually, `:system` breaks things in CI:

```
** (Protocol.UndefinedError) protocol List.Chars not implemented for {:system, "POSTGRES_HOSTNAME"} of type Tuple
```

(e.g. https://semaphoreci.com/mbta/skate/branches/pwd-postgres-docs/builds/39)